cli/cmd/vulnerability.go

```diff
@@ -364,13 +394,51 @@ func pollScanStatus(requestID string) error {
 
 		cli.StopProgress()
 		cli.OutputHuman(buildVulnerabilityReportTable(assessment))
+
 		if vulCmdState.Html {
 			return generateVulnAssessmentHTML(assessment)
 		}
+
+		if vulFailureFlagsEnabled() {
+			vulnAssess := vulAssessment{
+				severityRating:        assessment.HighestSeverity(),
+				fixableSeverityRating: assessment.HighestFixableSeverity(),
+				fixableVulCount:       assessment.FixableVulnerabilities,
+			}
+			vulnAssess.validate()
+		}
+
 		return nil
 	}
 }
 
+type vulAssessment struct {
+	severityRating        string
+	fixableSeverityRating string
+	fixableVulCount       int32
+}
+
+func (assessment *vulAssessment) validate() {
+	severityRating, _ := eventSeverityToProperTypes(assessment.severityRating)
+	fixableSeverityRating, _ := eventSeverityToProperTypes(assessment.fixableSeverityRating)
+	threshold, _ := eventSeverityToProperTypes(vulCmdState.FailOnSeverity)
+
+	if vulCmdState.FailOnSeverity == "" && vulCmdState.FailOnFixable && assessment.fixableVulCount > 0 {
+		cli.OutputHuman("\nFixable vulnerabilities found. Exiting with error code %d\n", 9)
```

Comment:
Choosing to exit with `9` is fine. Though, we had already an idea/feature-request that suggests we should exit with different codes depending on the severity of the vulnerabilities found: https://github.com/lacework/go-sdk/issues/211

Maybe a quick example/idea would be to:
* exit with code `9` if we found `critical` vulnerabilities
* exit with code `8` if we found `high` vulnerabilities
* exit with code `7` if we found `medium` vulnerabilities
* exit with code `6` if we found `low` vulnerabilities

Reply:
Mmm, i'm not sure on this. With `fail_on_severity `flag the user is providing the severity level already. So they should assume if they input `critical` the failure was due to a critical failing being found. Happy to discuss this more though.

Reply:
Another point for discussion is to distinguish the `fail_on_severity` and `fail_on_fixable` with different error codes. Perhaps instead on different codes per vuln severity type.

Reply:
The team agreed that this level of error code is too granular for our first iteration of this feature. We will stick to a single error code that is not `1`.